ERROR - 2024-10-22 01:16:51 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\cafe\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\cafe\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'cafe', 3306, '', 0)
#1 C:\cafe\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\cafe\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\cafe\system\Database\BaseBuilder.php(1673): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#4 C:\cafe\system\Model.php(930): CodeIgniter\Database\BaseBuilder->countAll()
#5 C:\cafe\app\Controllers\DashboardController.php(22): CodeIgniter\Model->__call('countAll', Array)
#6 C:\cafe\system\CodeIgniter.php(933): App\Controllers\DashboardController->index()
#7 C:\cafe\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DashboardController))
#8 C:\cafe\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\cafe\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\cafe\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\cafe\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\cafe\system\rewrite.php(44): require_once('C:\\cafe\\public\\...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\cafe\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\cafe\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\cafe\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\cafe\system\Database\BaseBuilder.php(1673): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', NULL, false)
#3 C:\cafe\system\Model.php(930): CodeIgniter\Database\BaseBuilder->countAll()
#4 C:\cafe\app\Controllers\DashboardController.php(22): CodeIgniter\Model->__call('countAll', Array)
#5 C:\cafe\system\CodeIgniter.php(933): App\Controllers\DashboardController->index()
#6 C:\cafe\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DashboardController))
#7 C:\cafe\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\cafe\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\cafe\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\cafe\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 C:\cafe\system\rewrite.php(44): require_once('C:\\cafe\\public\\...')
#12 {main}
CRITICAL - 2024-10-22 01:16:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: dashboard]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1673): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows` FROM `pelanggan`', null, false)
 3 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->countAll()
 4 APPPATH\Controllers\DashboardController.php(22): CodeIgniter\Model->__call('countAll', [])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\DashboardController->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DashboardController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\cafe\\public\\index.php')
ERROR - 2024-10-22 02:08:27 --> mysqli_sql_exception: Duplicate entry '2' for key 'nomor_meja' in C:\xampp\htdocs\kantinUSN\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\kantinUSN\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `meja` S...', 0)
#1 C:\xampp\htdocs\kantinUSN\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `meja` S...')
#2 C:\xampp\htdocs\kantinUSN\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `meja` S...')
#3 C:\xampp\htdocs\kantinUSN\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `meja` S...', Array, false)
#4 C:\xampp\htdocs\kantinUSN\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\kantinUSN\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\kantinUSN\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\kantinUSN\app\Controllers\MejaController.php(42): CodeIgniter\Model->update('4', Array)
#8 C:\xampp\htdocs\kantinUSN\system\CodeIgniter.php(933): App\Controllers\MejaController->update('4')
#9 C:\xampp\htdocs\kantinUSN\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
#10 C:\xampp\htdocs\kantinUSN\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\kantinUSN\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\kantinUSN\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\kantinUSN\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\kantinUSN\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-10-22 02:08:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'nomor_meja'
[Method: POST, Route: meja/update/4]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `meja` SET `nomor_meja` = :nomor_meja:, `kapasitas` = :kapasitas:, `status` = :status:
WHERE `meja`.`id` IN :meja.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\MejaController.php(42): CodeIgniter\Model->update('4', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MejaController->update('4')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\kantinUSN\\public\\index.php')
CRITICAL - 2024-10-22 02:08:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'nomor_meja'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `meja` SET `nomor_meja` = \'2\', `kapasitas` = \'10\', `status` = \'tersedia\'
WHERE `meja`.`id` IN (\'4\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `meja` SET `nomor_meja` = \'2\', `kapasitas` = \'10\', `status` = \'tersedia\'
WHERE `meja`.`id` IN (\'4\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `meja` SET `nomor_meja` = :nomor_meja:, `kapasitas` = :kapasitas:, `status` = :status:
WHERE `meja`.`id` IN :meja.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\MejaController.php(42): CodeIgniter\Model->update('4', [...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MejaController->update('4')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\kantinUSN\\public\\index.php')
CRITICAL - 2024-10-22 02:08:27 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'nomor_meja'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `meja` SET `nomor_meja` = \'2\', `kapasitas` = \'10\', `status` = \'tersedia\'
WHERE `meja`.`id` IN (\'4\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `meja` SET `nomor_meja` = \'2\', `kapasitas` = \'10\', `status` = \'tersedia\'
WHERE `meja`.`id` IN (\'4\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `meja` SET `nomor_meja` = \'2\', `kapasitas` = \'10\', `status` = \'tersedia\'
WHERE `meja`.`id` IN (\'4\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `meja` SET `nomor_meja` = :nomor_meja:, `kapasitas` = :kapasitas:, `status` = :status:
WHERE `meja`.`id` IN :meja.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\MejaController.php(42): CodeIgniter\Model->update('4', [...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MejaController->update('4')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\kantinUSN\\public\\index.php')
ERROR - 2024-10-22 02:11:39 --> mysqli_sql_exception: Duplicate entry '5' for key 'nomor_meja' in C:\xampp\htdocs\kantinUSN\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\kantinUSN\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `me...', 0)
#1 C:\xampp\htdocs\kantinUSN\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `me...')
#2 C:\xampp\htdocs\kantinUSN\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `me...')
#3 C:\xampp\htdocs\kantinUSN\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `me...', Array, false)
#4 C:\xampp\htdocs\kantinUSN\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\kantinUSN\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\kantinUSN\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\kantinUSN\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\kantinUSN\app\Controllers\MejaController.php(24): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\kantinUSN\system\CodeIgniter.php(933): App\Controllers\MejaController->store()
#10 C:\xampp\htdocs\kantinUSN\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
#11 C:\xampp\htdocs\kantinUSN\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\kantinUSN\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\kantinUSN\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\kantinUSN\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\kantinUSN\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2024-10-22 02:11:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '5' for key 'nomor_meja'
[Method: POST, Route: meja/store]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (:nomor_meja:, :kapasitas:, :status:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\MejaController.php(24): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MejaController->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\kantinUSN\\public\\index.php')
CRITICAL - 2024-10-22 02:11:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '5' for key 'nomor_meja'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (\'5\', \'9\', \'tersedia\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (\'5\', \'9\', \'tersedia\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (:nomor_meja:, :kapasitas:, :status:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\MejaController.php(24): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MejaController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\kantinUSN\\public\\index.php')
CRITICAL - 2024-10-22 02:11:39 --> [Caused by] mysqli_sql_exception: Duplicate entry '5' for key 'nomor_meja'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (\'5\', \'9\', \'tersedia\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (\'5\', \'9\', \'tersedia\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (\'5\', \'9\', \'tersedia\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `meja` (`nomor_meja`, `kapasitas`, `status`) VALUES (:nomor_meja:, :kapasitas:, :status:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\MejaController.php(24): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MejaController->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MejaController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\kantinUSN\\public\\index.php')
